### ISSUE TITLE
Missing keys in running `process_mesh_for_conv_generalized_fullres.py`

### ISSUE THREAD

**Issue by qiminchen** [nihalsid/CADTextures#3]
Hi, I was able to run the processing code and generate the `.pt` file, however, a couple of keys are missing when saving the `.pt` files compared to the data you use to train Texturify.

These are the keys from your training data,
```
dict_keys(['input_positions', 'input_normals', 'input_laplacian', 'input_ff1', 'input_ff2', 'input_gcurv', 
'input_mcurv', 'target_colors', 'pool_locations', 'conv_data', 'pos_data', 'vertices', 'indices', 'normals', 
'ff2_data', 'lateral_maps'])
```

Below are the keys saved by `process_mesh_for_conv_generalized_fullres.py`
```
dict_keys(['input_positions', 'input_normals', 'input_laplacian', 'input_ff1', 'input_ff2', 'input_gcurv', 
'input_mcurv', 'target_colors', 'pool_locations', 'conv_data'])
```
https://github.com/nihalsid/CADTextures/blob/92e1339be9ec26edddc383697dbc9f848ecc65df/data_processing/process_mesh_for_conv_generalized_fullres.py#L115-L128

Basically `pos_data`, `vertices`, `indices`, `normals`, `ff2_data`, `lateral_maps` are missing, can you please help with this? Thanks in advance.
